### PR TITLE
feat: docs use cases

### PR DESCRIPTION
**Pull request by jeremylgn**
Self explanatory.

### CODE REVIEW

File: docs/whitepaper/use cases.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+This holistic approach fosters trust among stakeholders and encourages a widespread knowledge exchange.
+
+The scalability of the OKP4 protocol ensures it evolves with regulatory changes and embraces new data sources. It becomes the cornerstone of a sustainable future where environmental responsibility meets technological innovation.
+
+Interoperability ceases to be a concern as the protocol effortlessly integrates complex workflows and eliminates compatibility issues.
+
+### [Market](https://www.globenewswire.com/en/news-release/2023/04/18/2648655/0/en/Carbon-Offset-Carbon-Credit-Market-Size-Projected-to-Reach-1-602-7-Billion-by-2028-MarketsandMarkets.html)
+
+The global Carbon Credit Market is projected to grow remarkably, from USD **414.8 billion in 2023 to USD 1,602.7 billion by 2028, with a staggering CAGR of 31.0%.** This exponential growth reflects the growing recognition of the importance of carbon offsetting in combating climate change and achieving net-zero greenhouse gas emissions.
```
Comment (ccamel) [inline docs/whitepaper/use cases.md:38]: IMHO, placing a link directly within a title is uncommon and, I guess, not considered a good practice in Markdown. I would position the title in the paragraph directly.

```suggestion
### Market

The global [Carbon Credit Market](https://www.globenewswire.com/en/news-release/2023/04/18/2648655/0/en/Carbon-Offset-Carbon-Credit-Market-Size-Projected-to-Reach-1-602-7-Billion-by-2028-MarketsandMarkets.html) is projected to grow remarkably, from USD **414.8 billion in 2023 to USD 1,602.7 billion by 2028, with a staggering CAGR of 31.0%.** This exponential growth reflects the growing recognition of the importance of carbon offsetting in combating climate change and achieving net-zero greenhouse gas emissions.
```

File: docs/whitepaper/use cases.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+The global Carbon Offset/Carbon Credit Market is poised for remarkable growth in the coming years. Increasing awareness, investments in carbon capture technologies, and the rise of impactful projects drive this market expansion. By participating in carbon offsetting, OKP4 is part of a global effort to combat climate change while creating a more sustainable and resilient future for all.
+
+### How does it work? Input & output
+
+***Typical workflow:***
+
+1. **Data Collection** from various sources (ERP, CSR, etc.), actors (suppliers, clients, etc.), and formats (IoT sensors, meters, energy management systems, etc.)
+2. **Aggregation and normalization** conversion services, standardization, quality, joining services)
+3. **Processing and calculation** (GHG Protocol, VCS Protocol, Gold Standard, reference data, conversion factors) under different categories (scope 1: direct emissions, scope 2: energy-related indirect emissions, scope 3: upstream and downstream indirect emissions)
```
Comment (ccamel) [inline docs/whitepaper/use cases.md:48]:
```suggestion
1. **Data Collection** from various sources (ERP, CSR, etc.), actors (suppliers, clients, etc.), and formats (IoT sensors, meters, energy management systems, etc.).
2. **Aggregation and normalization** conversion services, standardization, quality, joining services).
3. **Processing and calculation** (GHG Protocol, VCS Protocol, Gold Standard, reference data, conversion factors) under different categories (scope 1: direct emissions, scope 2: energy-related indirect emissions, scope 3: upstream and downstream indirect emissions).
```

File: docs/whitepaper/use cases.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+ OpenAI Inc. offers grants "[to fund experiments in setting up a democratic process for deciding what rules AI systems should follow](https://openai.com/blog/democratic-inputs-to-ai)". OKP4 protocol can guarantee the storage and enforcement of the defined rules.
+
+### [Market](https://www.statista.com/statistics/1365145/artificial-intelligence-market-size/#:~:text=According%20to%20Next%20Move%20Strategy,a%20vast%20number%20of%20industries)
+
+Transparency issues apply to the whole field of AI. According to [Next Move Strategy Consulting](https://www.nextmsc.com/), the artificial intelligence market is expected to grow strongly in the coming decade. Its nearly $100 Billion U.S. value is expected to grow twentyfold by 2030, up to almost two trillion U.S. dollars. The AI market covers a vast number of industries. Everything from supply chains, marketing, product making, research, analysis, and more are fields that will, in some aspect, adopt artificial intelligence within their business.
```
Comment (ccamel) [inline docs/whitepaper/use cases.md:85]: Same here for the link in the title.

Suggestion:

```suggestion
### Market

Transparency issues apply to the whole field of AI. According to [Next Move Strategy Consulting](https://www.nextmsc.com/), the [artificial intelligence market](https://www.statista.com/statistics/1365145/artificial-intelligence-market-size/#:~:text=According%20to%20Next%20Move%20Strategy,a%20vast%20number%20of%20industries) is expected to grow strongly in the coming decade. Its nearly $100 Billion U.S. value is expected to grow twentyfold by 2030, up to almost two trillion U.S. dollars. The AI market covers a vast number of industries. Everything from supply chains, marketing, product making, research, analysis, and more are fields that will, in some aspect, adopt artificial intelligence within their business.
```

File: docs/whitepaper/use cases.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+
+### How it works, input & output
+
+Any entity provides any source of data to train any model.
+Let's take an example. Multiple media publishers could submit their newsfeeds to the protocol.
+ A Machine Learning company could use specifically some of these stream data to propose a new summary. Everyone could know from which publishers the summary is from. For each summary consumption, revenue sharing is applied between every concerned publisher.
+
+Example :
```
Comment (ccamel) [inline docs/whitepaper/use cases.md:93]:
```suggestion
*Example:*
```

File: docs/whitepaper/use cases.md [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+
+### [Market](https://www.statista.com/statistics/1365145/artificial-intelligence-market-size/#:~:text=According%20to%20Next%20Move%20Strategy,a%20vast%20number%20of%20industries)
+
+Transparency issues apply to the whole field of AI. According to [Next Move Strategy Consulting](https://www.nextmsc.com/), the artificial intelligence market is expected to grow strongly in the coming decade. Its nearly $100 Billion U.S. value is expected to grow twentyfold by 2030, up to almost two trillion U.S. dollars. The AI market covers a vast number of industries. Everything from supply chains, marketing, product making, research, analysis, and more are fields that will, in some aspect, adopt artificial intelligence within their business.
+
+### How it works, input & output
+
+Any entity provides any source of data to train any model.
+Let's take an example. Multiple media publishers could submit their newsfeeds to the protocol.
+ A Machine Learning company could use specifically some of these stream data to propose a new summary. Everyone could know from which publishers the summary is from. For each summary consumption, revenue sharing is applied between every concerned publisher.
```
Comment (ccamel) [inline docs/whitepaper/use cases.md:91]:
```suggestion

A Machine Learning company could use specifically some of these stream data to propose a new summary. Everyone could know from which publishers the summary is from. For each summary consumption, revenue sharing is applied between every concerned publisher.
```

File: docs/whitepaper/use cases.md [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+
+### Storytelling
+
+With OKP4, organizations finally have a solution for improving their governance management. Let's take the example of a D.A.O.: OKP4 will enable more precise governance closer to its members by optimizing the operational functioning of the D.A.O. Information monitoring, analysis of members' needs, monitoring of voting trends and projections, distribution of powers, the automated release of funds, etc. all these functionalities will help to streamline the decision-making process of organizations (decentralized or otherwise).
+
+### [Market](https://daotimes.com/6-interesting-dao-insights-from-2022/)
+
+Estimating the market size on a broad subject, such as organizations and communities, is difficult. Instead, let's deep dive into Decentralized Autonomous Organization (D.A.O.). As of January 2023, the total market capitalization of the D.A.O. market stood at $8.8 billion. The total number of decentralized organizations stood at an impressive 10,752, up from 4,830 organizations in three months.
```
Comment (ccamel) [inline docs/whitepaper/use cases.md:125]: Same here for the link in the title.

Suggestion:

```suggestion
### Market

Estimating the market size on a broad subject, such as organizations and communities, is difficult. Instead, let's deep dive into Decentralized Autonomous Organization (D.A.O.). As of January 2023, the total market capitalization of the [D.A.O. market](https://daotimes.com/6-interesting-dao-insights-from-2022/) stood at $8.8 billion. The total number of decentralized organizations stood at an impressive 10,752, up from 4,830 organizations in three months.
```

File: docs/whitepaper/use cases.md [hunk 7]
```diff
@@ -0,0 +1,182 @@
...
+
+### How it works, input & output
+
+***Example of a DEX algorithmic engine update.***
+
+A working group mandated by a community initiates discussions on a forum to improve the algorithmic remuneration model of a DEX to optimize the distribution of management fees to its clients. The group proposes a few avenues to explore and a budget envelope. Preliminary discussions take place on the forum and cross-cutting discussions via e-mail or online messaging (Telegram, WhatsApp or other).
+
+Thanks to the protocol, the group will formalize and build an initial workflow enabling a tool for analyzing and synthesizing discussions referenced in the Dataverse. The first stage of collecting discussions is implemented. Then these are aggregated and processed using text analysis and natural language processing tools according to the needs of the working group. Then, based on this analysis, a summary is created to facilitate understanding of all this information and made accessible to all members,
```
Comment (ccamel) [inline docs/whitepaper/use cases.md:133]:
```suggestion
Thanks to the protocol, the group will formalize and build an initial workflow enabling a tool for analyzing and synthesizing discussions referenced in the Dataverse. The first stage of collecting discussions is implemented. Then these are aggregated and processed using text analysis and natural language processing tools according to the needs of the working group. Then, based on this analysis, a summary is created to facilitate understanding of all this information and made accessible to all members.
```

File: docs/whitepaper/use cases.md [hunk 8]
```diff
@@ -0,0 +1,182 @@
...
+
+Thanks to OKP4, the working group and the wider community can ensure a clearer understanding of the discussions around proposals, thus promoting more informed and transparent decision-making. Following these discussions, a vote is opened to propose an external service provider to improve the DEX engine. Within this framework, a specialized "FinTech" company is mobilized to implement its analysis models and a customized expert report for an amount previously validated by vote.
+
+If the vote results are positive, then part of the funds are automatically released, and a project manager is identified to report on the proper execution of the work.
+
+The service provider then carries out the work, and once it has been completed and validated, the remaining funds are released to the service provider, and a report is generated detailing the entire decision-making process and the project's progress.
+
+## 4. Other use cases with OKP4
+
+**IoT and PoPW network as a Service**
```
Comment (ccamel) [inline docs/whitepaper/use cases.md:143]:
```suggestion
### IoT and PoPW network as a Service
```

File: docs/whitepaper/use cases.md [hunk 9]
```diff
@@ -0,0 +1,182 @@
...
+
+- **Environmental sensor network:** Manage a decentralized network for surveillance of air, water, noise, and other environmental factors.
+- **Driving data:** Decentralized sensor network between vehicles to enhance traffic safety and efficiency or collect actionable vehicle data.
+- **Communication network:** create a decentralized network for Wi-Fi/5G connections, or a decentralized network for satellite internet.
+- **Distributed energy resource network:** Connect batteries to the network to build a sustainable energy system.
+
+**Agriculture**
```
Comment (ccamel) [inline docs/whitepaper/use cases.md:150]:
```suggestion
### Agriculture
```

File: docs/whitepaper/use cases.md [hunk 10]
```diff
@@ -0,0 +1,182 @@
...
+
+- **Irrigation optimization:** Share soil sensor data and analysis models to optimize water use according to the specific needs of crops.
+- **Crop management by drone or satellite:** Share real-time data to improve decision-making, management, and thus enhance knowledge about crops.
+- **Agronomic research:** Share field data to support research on new cultivation methods and plant varieties.
+- **Sustainable soil management:** Share resources on soil quality and health to promote sustainable land management practices.
+- **Precision agriculture:** Share field sensor data and satellite images to enable more precise and efficient agriculture.
+- **Agricultural biodiversity:** Share observations on plant and animal species present in and around fields to monitor and support biodiversity.
+
+**Health**
```
Comment (ccamel) [inline docs/whitepaper/use cases.md:159]:
```suggestion
### Health
```

File: docs/whitepaper/use cases.md [hunk 11]
```diff
@@ -0,0 +1,182 @@
...
+
+- **Personalized medicine:** Share data and services for genome sequencing and patient lifestyle information to tailor treatments to the individual.
+- **Epidemiology:** Share public health data and analysis resources to monitor and understand the spread of diseases.
+- **Coordinated care:** Share patient medical records between different care providers for coordination and continuity of care.
+- **Cancer research:** Share imaging data, tumor gene expression profiles, and treatment responses to develop predictive models of cancer progression and treatment response.
+- **Disease prevention:** Share information on healthy behaviors and risk factors to prevent diseases.
+- **Telemedicine:** Share remote diagnostic tools and resources to improve access to care, especially in remote or underserved areas.
+
+**Industry**
```
Comment (ccamel) [inline docs/whitepaper/use cases.md:168]:
```suggestion
### Industry
```

File: docs/whitepaper/use cases.md [hunk 12]
```diff
@@ -0,0 +1,182 @@
...
+
+- **Predictive maintenance:** Share sensor data and analysis models to predict when equipment will need to be repaired or replaced.
+- **Supply chain optimization:** Share data on inventory, demand, and delivery times to improve supply chain efficiency.
+- **Product quality:** Share quality control data and analysis tools to improve product quality.
+- **Process simulation:** Share production data and simulation models to optimize manufacturing processes.
+- **Waste management:** Share data on waste production and tools to minimize waste production.
+- **Automation:** Share production data and machine learning models to automate certain production tasks.
+
+**AI**
```
Comment (ccamel) [inline docs/whitepaper/use cases.md:177]:
```suggestion
### AI
```

File: docs/whitepaper/use cases.md [hunk 13]
```diff
@@ -0,0 +1,182 @@
+---
+sidebar_position: 7
+---
+
+# Use cases
+
+## 1. OKP4: Pioneering a sustainable revolution in Carbon Credit Management
```
Comment (ccamel) [inline docs/whitepaper/use cases.md:7]: In the rest of the whitepaper, the titles are not numbered. I therefore propose not to do so here, for consistency's sake.

```suggestion
## OKP4: Pioneering a sustainable revolution in Carbon Credit Management
```

**Review by ccamel — CHANGES_REQUESTED** [review]
A few comments and suggestions ready to be incorporated. If it's okay for you, you add the "Suggested changes" in a batch and commit it.

File: docs/whitepaper/use cases.md [hunk 14]
```diff
@@ -0,0 +1,183 @@
...
+4. Individual D parameters its cloud environment, setting up how an orchestration service should store the execution request results.
+5. Individual D requests service execution with the ML Workflow from Corp C, using data from Corp A and Corp B. He submits a transaction with a Keplr wallet and pays with $KNOW tokens. The blockchain validates the execution request (access and execution authorizations).
+6. The orchestration service listens to the event from the blockchain. It recovers access keys and then executes the workflow training algorithm from Corp C, using Corp A & Corp B data it fetches.
+7. The orchestration service tracks the progress and state changes of all jobs within the workflow and reports this information to the blockchain. If all works well, service agreement rules are applied, and $KNOW tokens from Individual D are unlocked to Corp A, Corp B, et Corp C.
+8. The orchestration service stores the result in the provided Individual D's storage solution. Optionally, this new knowledge can also be referenced as a data source for other workflows. Individual D can have access to the newly generated knowledge.
```
Comment (antho31) [inline docs/whitepaper/use cases.md:107]:
```suggestion
A complete description of end-to-end ML training with the OKP4 protocol is available in [the technical part of the whitepaper](https://docs.okp4.network/whitepaper/architecture#connectem-all). 
```

**Review by antho31 — COMMENTED** [review]
Reference to the Architecture part suggestion

File: docs/whitepaper/use cases.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (JulienMassonnat) [reply docs/whitepaper/use cases.md]: Yes I think it's important that we explained the use case with an accurate workflow. It's not so technical, it's just not sexy (Corp A, Corp B, etc.).

[the remark this answers is not in the document]
Comment (antho31) [reply docs/whitepaper/use cases.md]: OK, so we have two distinct examples, one more "evasive" and another more detailed?

File: docs/whitepaper/use cases.md (hunk 13 above)
Comment (jeremylgn) [reply docs/whitepaper/use cases.md]: Done :heavy_check_mark:

File: docs/whitepaper/use cases.md (hunk 1 above)
Comment (jeremylgn) [reply docs/whitepaper/use cases.md]: Done :heavy_check_mark:

File: docs/whitepaper/use cases.md (hunk 2 above)
Comment (jeremylgn) [reply docs/whitepaper/use cases.md]: Done :heavy_check_mark:

File: docs/whitepaper/use cases.md [hunk 15]
```diff
@@ -0,0 +1,182 @@
...
+4. **Analysis and reporting** (additional analysis based on the usage of the results, detailed and automated reporting to identify significant sources, trends, and reduction opportunities). Providing access to public and economic actors.
+5. **Offsetting**: Carbon data sharing, verification, and registration in the official registry. This avoids double accounting of credits in parallel! The protocol enables continuous monitoring beyond point-in-time calculations to detect trends and implement measures.
+
+### And tomorrow?
+
+OKP4 protocol is promising, potentially becoming a widely adopted, standardized, and technologically advanced framework for carbon credit calculation and management. Its impact can extend beyond individual organizations, contributing to global efforts in combating climate change, achieving sustainable development goals worldwide, and ensuring fair revenue for participants. The protocol allows, at the same time, incentives organizations to share their better-quality data and create new business models.
+
+## 2. Transparent & Fair Model Training (Ml & AI)
```
[the remark this answers is not in the document]
Comment (jeremylgn) [reply docs/whitepaper/use cases.md]: Done :heavy_check_mark:

File: docs/whitepaper/use cases.md (hunk 3 above)
Comment (jeremylgn) [reply docs/whitepaper/use cases.md]: Done :heavy_check_mark:

**Review by ccamel — APPROVED** [review]
LGTM! Thanks :+1:

**Review by LucasPzn — APPROVED** [review]
Looks good to me 👍

**Review by ManonPoint26 — APPROVED** [review]

**Comment by ManonPoint26** [comment]
Yes 👍

**Review by antho31 — APPROVED** [review]

**Review by amimart — APPROVED** [review]

**Review by JulienMassonnat — APPROVED** [review]

**Pull request merged**